frontend/src/assets/styles/global.scss

```diff
@@ -1069,8 +1069,8 @@ cds-text-input::part(label)::after, .cds-text-input-required-label {
   color: var(--light-theme-text-text-error, #b32001);
```

Comment:
We need to change this class from ::after to ::before as well.
Otherwise the asterisk on **text-input**s will be black instead of read.

![image](https://github.com/bcgov/nr-forest-client/assets/79578735/cee42198-d4c1-434f-8086-8e161b0af7b6)

Reply:
It can't be after. I will text internally.

Reply:
Fixed